lib/generators/templates/models/account.rb

```diff
@@ -0,0 +1,14 @@
+class Account < BookingsyncPortal::Account
+  synced id_key: :uid, local_attributes: [:email]
+
+  # Uncomment to add Rails logger
+  # def api
+  #   @api ||= BookingSync::Engine::APIClient.new(token.token, logger: Rails.logger, account: self)
+  # end
+
+  # Uncomment to add Rails logger
+  # # Used to synchronize accounts using `Account.synchronize(remove: true)`
```

Comment:
I can confirm that it breaks cause of `synced_all_at` - I have removed the column from the table and it executes ok.
```
Account.synchronize(remove: true)
   (0.2ms)  BEGIN
  Account Load (0.3ms)  SELECT "accounts".* FROM "accounts" WHERE ("accounts"."uid" != 1)
  Account Load (0.2ms)  SELECT "accounts".* FROM "accounts" WHERE "accounts"."uid" = 1
   (0.2ms)  COMMIT
=> [#<Account:0x0000000455f6a0
  id: 1,
  created_at: Thu, 25 Jun 2015 16:19:35 UTC +00:00,
  updated_at: Thu, 25 Jun 2015 16:20:02 UTC +00:00,
  provider: "bookingsync",
  uid: 1,
  name: "Your Business Name",
  oauth_access_token:
   "35b74ab3795323dd4f195fade75f3a0ddd46749fbe402f66623517295257c0f3",
  oauth_refresh_token:
   "308aeebdfd5c3cee697b9150b17a95a9e0d9322bc0b0f674637902cabb06a05e",
  oauth_expires_at: "1435256361",
  email: "demo_en@bookingsync.com">]
```